src/metabase/models/database.clj

```diff
@@ -323,7 +324,8 @@
   [database]
   (-> database
       serdes/load-xform-basics
-      (update :creator_id serdes/*import-user*)))
+      (update :creator_id serdes/*import-user*)
+      (assoc :initial_sync_status "complete")))
```

Comment:
Since the import includes all of the entities generated by a sync for this database, we can consider its initial sync complete.